modules/mono/SCsub

```diff
@@ -60,3 +29,5 @@ if env["platform"] in ["osx", "iphone"]:
 
 if env["tools"]:
     env_mono.add_source_files(env.modules_sources, "editor/*.cpp")
+
+env_mono.Prepend(CPPPATH=["#modules/gdnative/include/"])
```

Comment:
Does this mean that the Mono module now depends on GDNative? It would be nice if it didn't, but if it does that's not a huge loss, as people using C# likely wouldn't mind some extra disk usage from GDNative if they're already dealing with the increased disk usage of the Mono module.

However, if it does, then SCons should prevent this combination from being attempted (in `config.py`, in `def can_build`, `return True` would have to be changed to `return env["module_gdnative_enabled"]`). Right now trying to build gives this output:

```
$ scons p=linux -j7 module_mono_enabled=yes module_gdnative_enabled=no
scons: Reading SConscript files ...
Enabling ALSA
Enabling PulseAudio
Enabling udev support
Mono root directory not found. Using pkg-config instead
Checking for C header file mntent.h... (cached) yes
scons: done reading SConscript files.
scons: Building targets ...
[ 96%] Linking Program        ==> bin/godot.linuxbsd.tools.64.mono
[ 99%] progress_finish(["progress_finish"], [])
[100%] /usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_method_bind_ptrcall':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:235: undefined reference to `godot_method_bind_ptrcall'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_method_bind_call':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:239: undefined reference to `godot_method_bind_call'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_string_name':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:245: undefined reference to `godot_variant_new_string_name'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_node_path':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:249: undefined reference to `godot_variant_new_node_path'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_object':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:253: undefined reference to `godot_variant_new_object'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_transform2d':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:257: undefined reference to `godot_variant_new_transform2d'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_basis':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:261: undefined reference to `godot_variant_new_basis'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_transform':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:265: undefined reference to `godot_variant_new_transform'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_aabb':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:269: undefined reference to `godot_variant_new_aabb'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_dictionary':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:273: undefined reference to `godot_variant_new_dictionary'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:277: undefined reference to `godot_variant_new_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_packed_byte_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:281: undefined reference to `godot_variant_new_packed_byte_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_packed_int32_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:285: undefined reference to `godot_variant_new_packed_int32_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_packed_int64_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:289: undefined reference to `godot_variant_new_packed_int64_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_packed_float32_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:293: undefined reference to `godot_variant_new_packed_float32_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_packed_float64_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:297: undefined reference to `godot_variant_new_packed_float64_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_packed_string_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:301: undefined reference to `godot_variant_new_packed_string_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_packed_vector2_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:305: undefined reference to `godot_variant_new_packed_vector2_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_packed_vector3_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:309: undefined reference to `godot_variant_new_packed_vector3_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_new_packed_color_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:313: undefined reference to `godot_variant_new_packed_color_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_bool':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:317: undefined reference to `godot_variant_as_bool'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_int':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:321: undefined reference to `godot_variant_as_int'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_float':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:325: undefined reference to `godot_variant_as_float'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_string':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:329: undefined reference to `godot_variant_as_string'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_vector2':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:333: undefined reference to `godot_variant_as_vector2'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_vector2i':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:337: undefined reference to `godot_variant_as_vector2i'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_rect2':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:341: undefined reference to `godot_variant_as_rect2'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_rect2i':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:345: undefined reference to `godot_variant_as_rect2i'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_vector3':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:349: undefined reference to `godot_variant_as_vector3'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_vector3i':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:353: undefined reference to `godot_variant_as_vector3i'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_transform2d':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:357: undefined reference to `godot_variant_as_transform2d'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_plane':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:361: undefined reference to `godot_variant_as_plane'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_quat':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:365: undefined reference to `godot_variant_as_quat'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_aabb':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:369: undefined reference to `godot_variant_as_aabb'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_basis':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:373: undefined reference to `godot_variant_as_basis'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_transform':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:377: undefined reference to `godot_variant_as_transform'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_color':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:381: undefined reference to `godot_variant_as_color'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_string_name':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:385: undefined reference to `godot_variant_as_string_name'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_node_path':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:389: undefined reference to `godot_variant_as_node_path'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_rid':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:393: undefined reference to `godot_variant_as_rid'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_callable':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:397: undefined reference to `godot_variant_as_callable'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_signal':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:401: undefined reference to `godot_variant_as_signal'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_dictionary':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:405: undefined reference to `godot_variant_as_dictionary'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:409: undefined reference to `godot_variant_as_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_packed_byte_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:413: undefined reference to `godot_variant_as_packed_byte_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_packed_int32_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:417: undefined reference to `godot_variant_as_packed_int32_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_packed_int64_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:421: undefined reference to `godot_variant_as_packed_int64_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_packed_float32_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:425: undefined reference to `godot_variant_as_packed_float32_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_packed_float64_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:429: undefined reference to `godot_variant_as_packed_float64_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_packed_string_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:433: undefined reference to `godot_variant_as_packed_string_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_packed_vector2_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:437: undefined reference to `godot_variant_as_packed_vector2_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_packed_vector3_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:441: undefined reference to `godot_variant_as_packed_vector3_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_as_packed_color_array':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:445: undefined reference to `godot_variant_as_packed_color_array'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_string_new_with_utf16_chars':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:451: undefined reference to `godot_string_new_with_utf16_chars'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_string_name_new_copy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:457: undefined reference to `godot_string_name_new_copy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_node_path_new_copy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:463: undefined reference to `godot_node_path_new_copy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_array_new_copy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:469: undefined reference to `godot_array_new_copy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_dictionary_new_copy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:475: undefined reference to `godot_dictionary_new_copy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_packed_byte_array_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:481: undefined reference to `godot_packed_byte_array_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_packed_int32_array_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:485: undefined reference to `godot_packed_int32_array_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_packed_int64_array_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:489: undefined reference to `godot_packed_int64_array_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_packed_float32_array_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:493: undefined reference to `godot_packed_float32_array_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_packed_float64_array_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:497: undefined reference to `godot_packed_float64_array_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_packed_string_array_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:501: undefined reference to `godot_packed_string_array_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_packed_vector2_array_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:505: undefined reference to `godot_packed_vector2_array_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_packed_vector3_array_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:509: undefined reference to `godot_packed_vector3_array_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_packed_color_array_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:513: undefined reference to `godot_packed_color_array_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_variant_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:517: undefined reference to `godot_variant_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_string_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:521: undefined reference to `godot_string_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_string_name_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:525: undefined reference to `godot_string_name_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_node_path_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:529: undefined reference to `godot_node_path_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_signal_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:533: undefined reference to `godot_signal_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_callable_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:537: undefined reference to `godot_callable_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_array_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:541: undefined reference to `godot_array_destroy'
/usr/bin/ld: modules/libmodule_mono.linuxbsd.tools.64.a(runtime_interop.linuxbsd.tools.64.o): in function `godotsharp_dictionary_destroy':
/home/aaronfranke/workspace/godot/modules/mono/glue/runtime_interop.cpp:545: undefined reference to `godot_dictionary_destroy'
collect2: error: ld returned 1 exit status
scons: *** [bin/godot.linuxbsd.tools.64.mono] Error 1
scons: building terminated because of errors.
```

Reply:
It does. We use their `godot_*` structs (Not sure if we actually need them. If it's only by pointer we could use Godot types directly), and to re-use some of their functions. It's nothing we couldn't re-write ourselves without too much effort, but since Godot already includes the GDNative module by default...
Good point on the build system though. I'll make it check that.